portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +17,32 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+ 
+var i = 0; 			// Start Point
+var images = [];	// Images Array
+var time = 3000;	// Time Between Switch
+	 
+// Image List
+images[0] = "./images/lavender.jpg";
+images[1] = "./images/school.jpg";
+images[2] = "./images/writing.jpg";
+
+// Change Image
+function changeImg(){
+	document.slide.src = images[i];
```

Comment:
Use document.getElementsByName or document.getElementById.  I'm not sure this will work in all browsers

Reply:
I tried changing it to document.getElementsByName and for some reason it stopped working

Reply:
I'm resolving because I tried to replace document.getElementsByName and it caused the functionality to stop working

Reply:
Thats probably because document.getElementsByNames returns an array instead of a single element.  You'd either have to loop over them or give the element a unique id and use document.getElementById